get_started/index.md

```diff
@@ -39,6 +39,7 @@ Run `julia` in the command line (or double-click the Julia executable) and now y
 ] add ReinforcementLearning
 
 using ReinforcementLearning
+using Flux, Random, BSON
```

Comment:
`Flux, Random, BSON` are dependencies of ReinforcementLearning.jl, so I think they will all be automatically precompiled when `using ReinforcementLearning`. Correct me if I'm wrong here.

Reply:
I had to precompile Flux, Random, BSON on julia-1.4 but not on julia1.5. I am not sure why